locales/de.json

```diff
@@ -1159,7 +1159,8 @@
 					"password": "Passwort: ",
 					"theEmailAddressEnables": "Die E-Mail-Adresse ermöglicht den Versand von Benachrichtigungen und erleichtert das Zurücksetzen des Passwortes.",
 					"weNeedFollowingDataForRegistration": "Für die Nutzung der {{shortTitle}} benötigen wir folgende Angaben:",
-					"weStoreUserDataSafely": "Die {{shortTitle}} bewahrt die Nutzerdaten sicher auf und gibt sie nicht an Dritte weiter. Die Verarbeitung der Daten erfolgt entsprechend der hohen gesetzlichen Datenschutz-Anforderungen."
+					"weStoreUserDataSafelyStudent": "Die {{shortTitle}} bewahrt die Nutzerdaten sicher auf und gibt sie nicht an Dritte weiter. Die Verarbeitung der Daten erfolgt entsprechend der hohen gesetzlichen Datenschutz-Anforderungen.",
+					"weStoreUserDataSafely": "Für die Nutzung der {{shortTitle}} bedarf es für Schüler unter {{age}} Jahren der Einwilligung eines Erziehungsberechtigten"
```

Comment:
weStoreUserDataSafely - key and value don't match

Reply:
what does it mean? I changed weStoreUserDataSafelyStudent back to weStoreUserDataSafely, and new one I named weStoreUserDataSafelyParent

Reply:
weStoreUserDataSafely has nothing to do with the content. the key describes something, which is then not available in terms of content.

Reply:
this would now apply to weStoreUserDataSafelyParent